src/urllib3/request.py

```diff
@@ -84,6 +86,16 @@ def request(
 
         urlopen_kw["request_url"] = url
 
+        if json is not None and body is not None:
+            raise TypeError(
+                "request got values for both 'json' and 'body', can only specify one. Prefer json"
```

Comment:
```suggestion
                "request got values for both 'json' and 'body', can only specify one."
```

IMO "Prefer json" is confusing. Probably best to cut it, or could add a link to https://urllib3.readthedocs.io/en/latest/user-guide.html (I don't think we do links in exceptions anywhere else yet, but I've grown to be a big fan of this in other projects).